CD Workflow failing

**Description**
The CD workflow present in this repo which is based on the official GitHub Actions documentation is failing when a new release is published with the following error:
`npm ERR! 401 Unauthorized - PUT https://registry.npmjs.org/@creativecommons%2fvocabulary - You must be logged in to publish packages.`
On searching the web for the possible reasons for this error, I found that since there are two package repositories, namely `npm` and `GitHub Packages` available, and also since Vocabulary is a scoped package, so both the `registry-url` and `scope` has to be explicitly specified in the workflow.

**Reproduction**
<!-- Steps to reproduce the behavior. -->
The error replicates the same behaviour as is observed by visiting [this link.](https://github.com/creativecommons/fonts/runs/622557954?check_suite_focus=true#step:5:197)

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->
The workflow shouldn't fail.

**Additional context**
<!-- Add any other context about the problem here. -->
The problem may be resolved by specifying the `registry-url` and the `scope` in the workflow. Same problem exists in the `vocabulary` repo and was being fixed there, but upon the suggestion of @dhruvkb, this issue will be first tried to be solved here as it may involve multiple iterations and this is relatively a low risk repo.

**Want to take up?**
<!-- Are you interested in tackling this issue yourself? -->
Yes

--- a/.github/workflows/cd.yml
+++ b/.github/workflows/cd.yml
@@ -20,6 +20,8 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: '12.16.1'
+        registry-url: https://registry.npmjs.org
+        scope: creativecommons
 
     - name: Install dependencies
       run: npm install --no-audit

